storm-server/src/main/java/org/apache/storm/LocalCluster.java

```diff
@@ -193,6 +194,10 @@ private LocalCluster(Builder builder) throws Exception {
             this.supervisors = new ArrayList<>();
             TmpPath nimbusTmp = new TmpPath();
             this.tmpDirs.add(nimbusTmp);
+            stormHomeBackup = System.getProperty(ConfigUtils.STORM_HOME);
```

Comment:
I'm not 100% sure that we need to use temporary directory for storm home while launching local cluster. Did you find any issue regarding this?

Reply:
The local cluster starts up Nimbus, which tries to start RockDbStore, which reads storm.home. So far it's been erroring out, but because Nimbus swallows the error it hasn't been an issue, apart from the meaningless error logs. I figure we might as well set storm.home when using a local cluster, since it's always set in a real daemon, and that's the environment we're trying to simulate.

Reply:
Thanks for the details. Now I can understand the issue clearly.